src/main/java/cardibuddy/logic/commands/SearchCardCommand.java

```diff
@@ -0,0 +1,59 @@
+package cardibuddy.logic.commands;
+
+import static cardibuddy.commons.core.Messages.MESSAGE_NOT_IN_DECK;
+import static java.util.Objects.requireNonNull;
+
+import cardibuddy.commons.core.Messages;
+import cardibuddy.logic.LogicToUiManager;
+import cardibuddy.model.Model;
+import cardibuddy.model.deck.exceptions.NotInDeckException;
+import cardibuddy.model.flashcard.SearchCardKeywordsPredicate;
+
+
+/**
+ * Finds and lists all decks and cards in cardibuddy whose title contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class SearchCardCommand extends Command {
+
+    public static final String COMMAND_WORD = "searchcard";
```

Comment:
do we want our command word to be like this though? cos it's a bit inconsistent
why not search card?

Reply:
changed to "search card" instead!